src/System.Transactions/src/System.Transactions.csproj

```diff
@@ -46,18 +46,12 @@
     <Compile Include="System\Transactions\Configuration\ConfigurationStrings.cs" />
     <Compile Include="System\Transactions\Configuration\DefaultSettingsSection.cs" />
     <Compile Include="System\Transactions\Configuration\MachineSettingsSection.cs" />
-    <Compile Include="System\Transactions\Trace\TraceRecord.cs" />
```

Comment:
>e\TraceRecord.cs" / [](start = 46, length = 19)

I am wondering if we can keep these files and "hide" the usage of ETW inside these methods. This might prevent the need for changing all of the scattered locations in the code where the tracing is produced.